src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,41 @@
+package duke.command;
```

Comment:
Good, using different packages for different usage of classes

Reply:
Thanks!